llvm/docs/GettingInvolved.rst

```diff
@@ -388,27 +384,18 @@ Guidance for office hours hosts
   from the list above.
 
 
-.. _IRC:
-
-IRC
----
+Discord
```

Comment:
You've removed the thing that lets you link to this section, if I'm recalling my RST correctly.